test/conformance/gen-specsummaries.sh

```diff
@@ -29,4 +29,6 @@ cd "${KUBE_ROOT}"
 hack/make-rules/build.sh vendor/github.com/onsi/ginkgo/v2/ginkgo test/e2e/e2e.test
 
 # dump spec
-./_output/bin/ginkgo --dryRun=true --focus='[Conformance]' ./_output/bin/e2e.test -- --spec-dump "${KUBE_ROOT}/_output/specsummaries.json" > /dev/null
+# TODO: update the focus string when the new Ginkgo release for dumping spec with dry run mode is available.
```

Comment:
Is there a Ginkgo issue for tracking this missing feature? Can we reference it here?

Reply:
Ah, found it in the commit message: https://github.com/onsi/ginkgo/issues/935

Would be good to include in the comment, too.

Reply:
we should file an issue if we do merge this, to keep track.

Reply:
I have already left the issue link at the line below.

Reply:
Please give others a chance to comment before closing a conversion as "resolved".

I believe this one is still open because I suspect that @BenTheElder wanted a tracking issue for this TODO in the Kubernetes repo...

Reply:
TODOs in the Kubernetes source code are not enough. They tend to get forgotten because there is no systematic tracking of them.

Reply:
pls see: https://github.com/kubernetes/kubernetes/issues/109744

Reply:
this has been fixed in Ginkgo 2.1.4 but we must use `dry-run` instead, will bump to Ginkgo 2.1.4 and remove the TODO here.